restcomm/restcomm.interpreter/src/main/java/org/restcomm/connect/interpreter/rcml/AttributeValidator.java

```diff
@@ -0,0 +1,548 @@
+package org.restcomm.connect.interpreter.rcml;
+
+
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.apache.log4j.Logger;
+
+public interface AttributeValidator {
+
+	public boolean RecordValidator (Tag tag);

```

Comment:
this interface should provide single API for any validator,instead of specific API per RCML verb,that is resolved later using 
[polimorphism](https://stackoverflow.com/questions/1031273/what-is-polymorphism-what-is-it-for-and-how-is-it-used)